stingray/stats.py

```diff
@@ -0,0 +1,268 @@
+import warnings
+from collections.abc import Iterable
+
+import numpy as np
+from scipy import stats
+
+
+__all__ = ['p_multitrial_from_single_trial',
+           'p_single_trial_from_p_multitrial',
+           'fold_profile_probability',
+           'fold_detection_level',
+           'z2_n_detection_level',
+           'z2_n_probability']
+
+
+def p_multitrial_from_single_trial(p1, n):
+    """Calculate a multi-trial probability from a single-trial one.
```

Comment:
Needs precise definitions in the docstring for what `p1` and `n` are.

Reply:
Also: how is this related to the `classical_pvalue` implemented for the PSD? Should that function there be entirely replaced by this one? If so, we should do that. What I don't understand is how this relates to the formalism in Groth (1975), which computes the probabilities for single-trial probabilities (this is what's implemented in `classical_pvalue`)?

Reply:
No, this is not related to that. This is a generic function that, given a single trial probability of p1, gives the probability of at least one success in n trials.

This substitutes the epsilon * n I formerly used in the calculation of the power probabilities, which was a good approximation if epsilon * n << 1, but not as this quantity approaches (or even surpasses) 1.